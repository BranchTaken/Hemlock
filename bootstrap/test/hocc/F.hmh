hocc
    token A
    token B
    token C
    token D
    token E

    start S ::=
      | A M D
      | B N D
      | A N E
      | B M E

    nonterm M ::= C

    nonterm N ::= C
