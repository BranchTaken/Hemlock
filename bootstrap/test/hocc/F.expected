hocc: Parsing "./F.hmh"
hocc: Generating LR(1) specification
hocc: 0 precedences, 7 tokens, 4 non-terminals (1 start), 7 productions
hocc: LR(1) item set compatibility: lr1
hocc: Generating LR(1) item set closures (+^.=add/split/merge)++++++^+++++++
hocc: Generating 15 LR(1) states
hocc: 0 unresolvable conflicts in 0 states
hocc: 0 unreachable states
hocc: 0 remergeable states
hocc: Searching for unused precedences/tokens/non-terminals/productions
hocc: Generating text report
hocc: Generating hocc report
hocc: Writing "./hocc/F.txt"
hocc: Writing "./hocc/F.hmh"
